support/modules/rootfs-image

```diff
@@ -2,10 +2,13 @@
 
 set -ue
 
-# Is expected to contain two variable definitions with device paths:
-# - MENDER_ROOTFS_PART_A
-# - MENDER_ROOTFS_PART_B
-. /etc/mender/rootfs-image-v2.conf
+STATE="$1"
+FILES="$2"
+
+CONF_FILE="/var/lib/mender/mender.conf"
```

Comment:
Q: Do we not support overriding `/var/lib/` with an environment variable?

Also, you're not gonna agree here, but in _theory_, this could be overridden in the local config file

Reply:
Yes, but we didn't support this in the old client, so I don't think we should start adding it now. At least it's not something to prioritize here.

Reply:
Since we're now renaming it, is it not the opportune moment?

Reply:
Wait, are you talking about the path, or the config file name?

The config file name I hadn't even thought about... But yeah, in a new major release, it is indeed a good opportunity to rename it. However, this is a pretty big change, if we want to do it I think it's better to add a task for it and fix it later in the project.

Does it sound like a good idea to split the file into `mender-auth.conf` and `mender-update.conf`?

Reply:
> Does it sound like a good idea to split the file into mender-auth.conf and mender-update.conf?

Reply:
Principally I agree as well. I suspect it's quite a lot of work to do this though, due to all the moving pieces that interact with `mender.conf`. For now I have added it as a client team meeting topic.